src/jupyter/PopulateDatabase.ipynb

```diff
@@ -299,27 +302,27 @@
       "            \"category_id\": \"relocations\",\n",
       "            \"type\": \"flow\",\n",
       "            \"order\": 2,\n",
-      "            \"flowgeek_url\": \"https://www.flowgeek.org/from-data-to-insights/calculating-mobility-indicators/relocation-indicators/\",\n",
+      "            \"flowgeek_url\": \"https://www.flowgeek.org/methods/calculating-mobility-indicators/relocation-indicators/\",\n",
       "            \"label\": \"Relocation\",\n",
       "            \"description\": \"Relocation-class indicators describe long-term (monthly) changes in the number of people changing their home location between pairs of areas.\",\n",
       "            \"label_fr\": \"Changement de r\\u00e9sidence\",\n",
       "            \"description_fr\": \"Les indicateurs relatifs au changement de r\\u00e9sidence d\\u00e9crivent les variations (mensuelles) \\u00e0 long terme du nombre de personnes qui changent de lieu de r\\u00e9sidence entre des paires de zones.\"\n",
       "        },\n",
       "        {\n",
-      "            \"category_id\": \"visitors\",\n",
+      "            \"category_id\": \"presence\",\n",
       "            \"type\": \"single_location\",\n",
       "            \"order\": 3,\n",
-      "            \"flowgeek_url\": \"https://www.flowgeek.org/from-data-to-insights/calculating-mobility-indicators/presence-indicators/\",\n",
+      "            \"flowgeek_url\": \"https://www.flowgeek.org/methods/calculating-mobility-indicators/presence-indicators/\",\n",
```

Comment:
This is a cheeky unrelated fix, right?

Reply:
Yes. As I was debugging the ingestion (because some indicators were missing after the ingestion had finished) I found inconsistent naming so I fixed it all in one go.
Apologies!